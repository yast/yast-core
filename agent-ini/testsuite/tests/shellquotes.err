[Interpreter] tests/shellquotes.ycp:XXX Unquoting error: Missing enclosing double quote. Unquoted value: <"invalid - unclosed>.
