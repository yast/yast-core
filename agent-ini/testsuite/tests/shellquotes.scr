.

`ag_ini(
    `IniAgent( "tests/shellquotes.in.test",
	$[
	    /* values at the top level, allow comments after value */
	    "options"	: [ "global_values", "shell_quoted_value", "join_multiline" ],
	    /* empty lines, comments after value and whole-line comments */
	    "comments"	: [ "^[ \t]*$", "[ \t]*#.*$", "^[ \t]*#.*$" ],
	    /* matching rules */
	    "params"	: [
		/* VAR1="val1 'bal x" */
		$[ "match" : [ "^[ \t]*([a-zA-Z_][a-zA-Z0-9_]*)=(.*)", "%s=%s" ] ],
	    ],
	]
))
