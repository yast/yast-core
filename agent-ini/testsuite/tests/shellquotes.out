([])
Val1='string'
Val2='string1string2'
Val3='shell_quoted_value'\''s "speciality"'
Val5='string with space'
Val6='string with escaped spaces'
Val7='funny quotes'
Val8='string1string2'
Val9='squote '\''inside'\'' dquote'
Val10='dquote "inside" squote'
Val11='Damned Joe'\''s garage'
Val12='Tabbed\tstring'
Val4=''\''written'\'' "value"'
